plugins/swf-backend/src/service/router.ts

```diff
@@ -125,9 +125,23 @@ function setupInternalRoutes(
     const {
       params: { swfId },
     } = req;
-    const wsRequest = await fetch(
-      `${kogitoBaseUrl}:${kogitoPort}/management/processes/${swfId}/source`,
-    );
+
```

Comment:
I think it'd be nice to put this re-try on every call we make to the kogito container.

Anything the User does that could make Quarkus restart could run into same failure state.

WDYT?